src/polyswarmd/monkey.py

```diff
@@ -16,7 +18,7 @@ def patch_gevent():
 
 def patch_web3():
     def make_post_request(endpoint_uri, data, *args, **kwargs):
-        kwargs.setdefault('timeout', 1)
+        kwargs.setdefault('timeout', 10)
```

Comment:
Is 10 going to cause us to DoS ourselves again when IPFS URIs don't exist?

Reply:
This is only when posting to web3, not IPFS.